VirtoCommerce.Platform.Core/ModuleFileCollector/ModuleFile.cs

```diff
@@ -0,0 +1,9 @@
+namespace VirtoCommerce.Platform.Core.ModuleFileCollector
+{
+    public class ModuleFile
+    {
+        public string WebPath { get; set; }
+        public string Version { get; set; }
+        public bool IsVendor { get; set; }
```

Comment:
As I can see in `ModulesScriptsResolveTagHelper` and `ModulesStylesResolveTagHelper` classes, all files are treated same way regardless of the value of this property. Is it really necessary?

Reply:
What exactly property you mean?

Reply:
I'm talking about `IsVendor` property.

Reply:
But we need to include vendor files first, if there are. That's why we need this property.
![image](https://user-images.githubusercontent.com/15198683/50342291-183e9d00-052b-11e9-9833-03a5bfc488e5.png)